From 1aad113de6ce11e469bfd7945fa547ff28f06297 Mon Sep 17 00:00:00 2001
From: arian <arian.kulmer@web.de>
Date: Mon, 18 Feb 2019 17:23:31 +0100
Subject: [PATCH 4/4] sepolicy: Add core_data_file_type type to cnd_data_file

* The following types on /data/ must be associated with the "core_data_file_type" attribute: cnd_data_file

Change-Id: I523f4845953232492561beda98c938c8d727b352
---
 vendor/common/file.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/common/file.te b/vendor/common/file.te
index 8c747fe1..f4729d38 100644
--- a/vendor/common/file.te
+++ b/vendor/common/file.te
@@ -28,7 +28,7 @@ type qdcmsocket_socket, file_type;
 
 # Define cnd socket and data file type
 type cnd_socket, file_type, mlstrustedobject;
-type cnd_data_file, file_type, data_file_type;
+type cnd_data_file, file_type, data_file_type, core_data_file_type;
 type chre_socket, file_type;
 
 # Define dpmd data file type
-- 
2.17.1

